modules/ipi-install-preparing-the-bare-metal-node.adoc

```diff
@@ -6,7 +6,8 @@
 [id='preparing-the-bare-metal-node_{context}']
 = Preparing the bare metal node
 
-Expanding the cluster requires a DHCP server. Each node must have a DHCP reservation.
+To expand your cluster, you must provide the node with the relevant IP address. This can be done with a static configuration, or with a DHCP server. When expanding the cluster using a DHCP server, each node must have a DHCP reservation.
```

Comment:
@kelbrown20  sorry, not sure I understood your comment. Can you elaborate?

Reply:
Yes! My apologies! Sometimes for acronyms, we write out what the acronym stands for in parenthesis, at least once somewhere in the doc. In this example it would be something like:

`or with a DHCP (Dynamic host Configuration protocol) server`

We don't do this for common abbreviations and I was asking if you thoughts on this one and if it should be expanded. 🙂

Reply:
Ah, got it thanks! This is not my usual documentation area, I was documenting a bug. I'll add this!